pkg/handler/handler.go

```diff
@@ -59,52 +60,52 @@ type timedSpec struct {
 // OpenAPIService is the service responsible for serving OpenAPI spec. It has
 // the ability to safely change the spec while serving it.
 type OpenAPIService struct {
-	specCache  cached.Replaceable[*spec.Swagger]
-	jsonCache  cached.Data[timedSpec]
-	protoCache cached.Data[timedSpec]
+	specCache  cached.LastSuccess[*spec.Swagger]
```

Comment:
What's our goal with the Replaceable interface? I noticed Atomic implements it, should LastSuccess also follow it?

Reply:
added a type assertion for `LastSuccess`. Done.